src/octoclass.cpp

```diff
@@ -1074,7 +1118,6 @@ void OctoClass::OccNodesWithinRadius(const geometry_msgs::Point &center_pt,
 
 bool OctoClass::NearestOccNodeWithinRadius(const geometry_msgs::Point &center_pt,
                                            const double &radius,
-                                           Eigen::Vector3d *node_center,
```

Comment:
I had noticed that this was not used anywhere, so I removed this output from this function